docs/prow/presets.md

```diff
@@ -0,0 +1,27 @@
+# Presets
+
+This document contains the list of all Presets available in the [`config.yaml`](../../prow/config.yaml). Use them to define ProwJobs for your components.
+
+| Name                               | Description                                                                                                                                                     |
+| ---------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **preset-dind-enabled**            | It allows Docker to run in your job.                                                                                                                        |
+| **preset-sa-gcr-push**             | It injects credentials for pushing images to Google Cloud Registry (GCR).                                                                                             |
+| **preset-docker-push-repository**  | It provides the environment variable with the location of the Docker repository.                                                                                 |
+| **preset-build-pr**                | It provides the environment variable with the location of the directory in the Docker repository for storing images. It also sets the **BUILD_TYPE** variable to `pr`. |
+| **preset-build-master**            | It is similar to the **preset-build-pr** Preset, but the **BUILD_TYPE** variable is set to `master`.                                                            |
+| **preset-build-release**           | It is similar to the **preset-build-pr** Preset, but the **BUILD_TYPE** variable is set to `release`.                                                           |
+| **preset-gc-project-env**          | It provides the environment variable with the Google Cloud Platform (GCP) project name.                                                                                              |
+| **preset-gc-compute-envs**         | It provides environment variables with the gcloud compute zone and the gcloud compute region.                                                                   |
```

Comment:
```suggestion
| **preset-gc-compute-envs**         | It provides environment variables with the GCP compute zone and the GCP compute region.                                                                   |
```
I used GCP as Google Cloud Platform was mentioned one row above, but you can also use the full name.